src/Institution/Institution.php

```diff
@@ -1,39 +1,127 @@
-<?php declare(strict_types=1);
+<?php
+
+declare(strict_types=1);
 
 namespace App\Institution;
 
+use App\Ontology\OpenSkos;
+use App\Ontology\VCard;
+use App\Rdf\Iri;
+use App\Rdf\Literal;
 use App\Rdf\Triple;
 
-final class Institution {
+final class Institution
+{
+    const code = 'code';
+    const name = 'name';
+    const organisationUnit = 'organisationUnit';
+    const email = 'email';
+    const website = 'website';
+    const streetAddress = 'streetAddress';
+    const locality = 'locality';
+    const postalCode = 'postalCode';
+    const countryName = 'countryName';
+    const enableStatusesSystem = 'enableStatusesSystem';
+    const enableSkosXl = 'enableSkosXl';
 
     /**
-     * @var Triple[]
+     * @var string[]
      */
-    private $triples = [];
+    private static $mapping = [
+        self::code => OpenSkos::CODE,
+        self::name => OpenSkos::NAME,
+        self::organisationUnit => VCard::ORGUNIT,
+        self::email => VCard::EMAIL,
+        self::website => OpenSkos::WEBPAGE,
+        self::streetAddress => VCard::ADR,
+        self::locality => VCard::LOCALITY,
+        self::postalCode => VCard::PCODE,
+        self::countryName => VCard::COUNTRY,
+        self::enableStatusesSystem => OpenSkos::ENABLESTATUSSESSYSTEMS,
+        self::enableSkosXl => OpenSkos::ENABLESKOSXL,
+    ];
 
-    private function __construct() {}
+    /**
+     * @var Iri
+     */
+    private $subject;
 
     /**
-     * @return string[]
+     * @var Literal[]
      */
-    public function properties() : array
+    private $literals = [];
+
+    public function __construct(Iri $subject)
     {
-        $res = [];
-        foreach ($this->triples as $triple) {
-            $res[$triple->getPredicate()] = $triple->getObject();
-        }
+        $this->subject = $subject;
+    }
+
+    //TODO: Generate getters.
```

Comment:
Todo can be removed.

Reply:
Nope :D Especially this one. I didn't generate all getters. only two of them.